versioned_docs/version-2.x/magento1/installation.mdx

```diff
@@ -217,9 +219,30 @@ For more security add a random key on `frontcommerce_secret_key` in your
     // ...
 ```
 
+### Configure facet order
+
+<SinceVersion tag="2.19" />
+
+If you want to be able to organize your facets' order, you will need to enable
+it in the `magentoProvider` (see
+[code](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/fcaf0d0ddec410caf1a4d250d197e5f382e9004b#d43f790dbbf8f5142f59364a4b017caa013c10b5_82_83)).
+In this provider, override `magento.attributes.useFacetSortOrder` to set it to
+`true`.
+
+:::tip
+
+To learn more about how to override configuration provider, see our
+[guide on overriding an existing provider](https://developers.front-commerce.com/docs/2.x/advanced/server/configurations#override-an-existing-configuration).
+
+:::
+
+Once that's done, all facets will be sorted by their "position" attribute that
+is defined in the Magento's back office.
+
 ### Clean Magento cache
 
 ## Ensure it works Once this is done, all the checks should be green in your
+
```

Comment:
this page content is bit broken (too much content in headings) but adding this line break makes it worse

Reply:
Yeah I'm not sure why there's a line break here. I think that should be 
```
## Ensure it works

Once this is done blblablab
```

Will fix.